src/posts/xarray-biology/index.md

```diff
@@ -0,0 +1,135 @@
+
+<RawHTML filePath='/posts/xarray-biology/dataset-repr.html' />
+
+Finally, the [`DataTree`](https://docs.xarray.dev/en/latest/user-guide/data-structures.html#dataset) object allows you to keep trees of related datasets, for example, a representation of multiscale images allows for natural support of [OME-NGFF](https://www.nature.com/articles/s41592-021-01326-w) data.
+
+<RawHTML filePath='/posts/xarray-biology/datatree-repr.html' />
+
+**Computational Patterns**
+
+Not only does this metadata and coordinates make the developer experience better, but they allow xarray to provide incredibly powerful basic [computational patterns](https://tutorial.xarray.dev/intermediate/01-high-level-computation-patterns.html) such as `groupby`, `coarsen`, `resample` and more. Having access to these tools can, with a little effort, mean you no longer need to write complicated for loops to do your data analysis. It will significantly speed up your ability to explore your data
+
+### When Can You Use Xarray?
+
+As great as Xarray sounds, it does have limitations. After all, Xarray is an Array library; it’s in the name! So if your data is tabular, Xarray is probably not the right tool for you. To get a better understanding of this, I recommend reading Ryan Abernathy’s excellent post [Tensors vs Tables](https://earthmover.io/blog/tensors-vs-tables?ref=xarray-bio-blog).
+
+That said, many parts of biology generate array data. And scientists in some areas are already benefiting from Xarray.
+
+### Who is Using Xarray
+
+- [`Sgkit`](https://sgkit-dev.github.io/sgkit/latest/getting_started.html#data-structures) “a toolkit for quantitative and population genetics” uses `Xarray` `Dataset`s as a core data structure. This allows them to keep track of the rich metadata associated with sequences and further uses computational patterns to speed up analyses.
+- `Xarray` has also been used for neurophysiology data by the Ecephys package. See a previous blog post [xarray for neurophysiology](https://xarray.dev/blog/xarray-for-neurophysiology) to learn more.
+- The `scverse` family of software uses `anndata`, which, like `Xarray` provides labeled dimensions.
+- The `bioio` package, capable of reading a zoo of microscope formats, provides a function to get image stacks data as Xarray object.
+- The `Xarray.DataTree` provides a natural representation of multiscale images and can interoperate well with the OME-NGFF model. The [multiscale-spatial-image](https://github.com/spatial-image/multiscale-spatial-image?tab=readme-ov-file#multiscale-spatial-image) project use DataTree for precisely this purpose.
+
+## What has limited adoption by Biologists?
+
+Given the benefits of switching to `Xarray`, why aren’t more biologists using it? Is it secretly not as good as this blog claims?
```

Comment:
I think you should put something like this up at the start. The thesis of this post is that
a) Xarray is a powerful tool for bio
b) Some bio people use it
c) A lot more bio people should use it
d) Ideas for how to make that happen

This thesis should be communicated early on and clearly.